runtime/basilisk/src/governance/old.rs

```diff
@@ -19,29 +19,12 @@ use crate::governance::TechnicalCollective;
 use crate::*;
 
 use primitives::constants::{
-	currency::{CENTS, DOLLARS, UNITS},
+	currency::DOLLARS,
```

Comment:
Why remove this and then use imports directly? Just change to primitives if it changed?